src/NServiceBus.RavenDB/Outbox/OutboxPersister.cs

```diff
@@ -109,16 +108,16 @@ public async Task SetAsDispatched(string messageId, ContextBag options, Cancella
                 if (useClusterWideTransactions)
                 {
                     var compareExchangeKey = $"rvn-atomic/{outboxRecordId}";
```

Comment:
Can't this get a better name? Could multiple of such keys exists? Maybe the name could contain the intent?
```suggestion
                    var outboxLockCompareExchangeKey = $"rvn-atomic/{outboxRecordId}";
```